### PR TITLE
add from component

### PR DESCRIPTION
**Pull request by MuhammadJaafer**

### CODE REVIEW

File: frontend/components/Login/Login.tsx
Comment (gauravjot) [inline frontend/components/Login/Login.tsx:1]: This seems like a template form instead of an actual login form.

**Review by gauravjot — COMMENTED** [review]
Looks good to me.

**Comment by MuhammadJaafer** [comment]
check it now, the login form is complete.

**Comment by MuhammadJaafer** [comment]
Sign up form ready

**Comment by MuhammadJaafer** [comment]
small edit the signup form button text was  login 
changed it to sign up

**Pull request merged**